flareon/src/test.rs

```diff
@@ -178,6 +186,15 @@ impl TestRequestBuilder {
             );
         }
 
+        #[cfg(feature = "json")]
+        if let Some(json_data) = &self.json_data {
+            *request.body_mut() = Body::fixed(json_data.clone());
+            request.headers_mut().insert(
+                http::header::CONTENT_TYPE,
+                http::HeaderValue::from_static("application/json"),
```

Comment:
Why not use `flareon::headers::JSON_CONTENT_TYPE`?

Reply:
Just a good practice not to use any data used by production code in tests. This might be overdoing stuff (after all, it's just a well-known constant), but this prevents tests from passing when, for instance, the constant would be refactored and modified by a mistake.